src/fights.ts

```diff
@@ -2247,8 +2248,24 @@ function yachtzee(): void {
 
       if (!equippedOutfit || !success()) return;
 
-      setChoice(918, 2);
+      const lastUMDDate = property.getString("umdLastObtained");
+      const today = Date.now() - gametimeToInt() - 1000 * 60 * 3.5; // Import today from ./lib once the PR is merged
+      const getUMD =
+        !get("_sleazeAirportToday") && // We cannot get the UMD with a one-day pass
+        garboValue($item`Ultimate Mind Destroyer`) >=
+          2000 * (1 + numericModifier("meat drop") / 100) &&
+        (!lastUMDDate || today - Date.parse(lastUMDDate) >= 1000 * 60 * 60 * 24 * 7);
+
+      if (getUMD) setChoice(918, 1);
+      else setChoice(918, 2);
```

Comment:
@docrostov may set my house on fire and say the exact opposite is true, but 
```suggestion
     setChoice(918, getUMD ? 1 : 2);
```
might be a little cleaner